src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -347,6 +347,13 @@ if(CLR_CMAKE_TARGET_SUNOS)
   )
 endif(CLR_CMAKE_TARGET_SUNOS)
 
+if (NOT HAVE_BUILTIN_GNUC_ATOMICS)
+  target_link_libraries(coreclrpal
+    PUBLIC
+    atomic
```

Comment:
If this is really needed, we would need to understand licensing and whether it is introducing any new pre-requisites.

Reply:
Googling around seems to indicate `libatomic` is required for 64bit atomics on some 32bit platforms, AFAIR it's licensed the same as `libgcc` (which we already depend on for most platforms) and both should be installed together on most systems.

I'm wondering why x86 seems to still fail with it though.